nbs/models.timemixer.ipynb

```diff
@@ -736,8 +729,8 @@
     "        futr_exog = windows_batch['futr_exog']\n",
     "\n",
     "        if self.futr_exog_size > 0:\n",
-    "            x_mark_enc = futr_exog[:,:self.input_size,:]\n",
-    "            x_mark_dec = futr_exog[:,-(self.label_len+self.h):,:]\n",
+    "            x_mark_enc = futr_exog[:, :, :self.input_size]\n",
+    "            x_mark_dec = futr_exog[:, -(self.h):, :]\n",
```

Comment:
This is incorrect, the shape of futr_exog is [B, F, L + h, N], so the selection in the sequence dimension should be in the third dimension.

Reply:
Fixed!